zerver/tests/test_signup.py

```diff
@@ -266,7 +266,7 @@ def test_register(self):
         with queries_captured() as queries:
             self.register("test@zulip.com", "test")
         # Ensure the number of queries we make is not O(streams)
-        self.assert_length(queries, 47)
+        self.assert_length(queries, 70)
```

Comment:
Looks like [this](https://github.com/zulip/zulip/pull/4156/commits/f1be02a4d965ee2bad7d0247f2d1ca7306cfdc11#r118293609) change is causing so many new database queries to be added. With only the `notifications_stream` addition in `populate_db.py` to master, this test fails.